core/controllers/editor_test.py

```diff
@@ -629,6 +629,10 @@ def test_exploration_download_handler_with_invalid_exploration_id(self):
 
         self.logout()
 
+    def test_guest_cannot_access_exploration_download_handler(self):
+        self.save_new_valid_exploration('exp_id', 'owner_id')
```

Comment:
Can we add a baseline verification? (maybe "Editor can download the exploration"?)

Reply:
There are tons of tests which checks for editor rights (in both acl_decorators and editor_test)